cocos/primitive/cylinder.ts

```diff
@@ -50,9 +48,9 @@ const temp2 = new Vec3(0, 0, 0);
  * but may be repositioned through the `center` option.
  * @zh
  * 生成一个圆柱。
- * @param radiusTop 顶部半径。
- * @param radiusBottom 底部半径。
- * @param opts 圆柱参数选项。
+ * @param radiusTop @en Radius of top plane. @zh 顶部半径。
+ * @param radiusBottom @en Radius of bottom plane.  @zh 底部半径。
+ * @param opts @en Options of cylinder.  @zh 圆柱参数选项。
  */
 export default function cylinder (radiusTop = 0.5, radiusBottom = 0.5, height = 2, opts: RecursivePartial<ICylinderOptions> = {}): IGeometry {
```

Comment:
Do you leak height param?

Reply:
Yep, i only modified exist comments. I will add it. 
Good catch.

Reply:
I fixed it.